src/vs/workbench/contrib/chat/common/promptSyntax/computeAutomaticInstructions.ts

```diff
@@ -88,6 +88,9 @@ export class ComputeAutomaticInstructions {
 		const telemetryEvent: InstructionsCollectionEvent = newInstructionsCollectionEvent();
 		const context = this._getContext(variables);
 
+		// add instructions from providers (always included)
+		await this.addProviderInstructions(instructionFiles, context, variables, telemetryEvent, token);
```

Comment:
There should be no special handling in computeAutomaticInstructions. The provider instructions are not any special. If we want to to be always included they should define `applyTo: **`